hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_unr_excl.el

```diff
@@ -174,521 +76,460 @@ ANNOTATION: "VC_COV_UNR"
 Block 23 "3305187838" "gen_normal_fifo.fifo_wptr <= (gen_normal_fifo.fifo_wptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
 ANNOTATION: "VC_COV_UNR"
 Block 32 "2130482102" "gen_normal_fifo.fifo_rptr <= (gen_normal_fifo.fifo_rptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
+CHECKSUM: "3042386013 188062951"
+INSTANCE: tb.dut.u_tlul_adapter_sram
+ANNOTATION: "VC_COV_UNR"
+Block 20 "3478134645" "d_valid = 1'b1;"
 CHECKSUM: "3839527590 2385261621"
 INSTANCE: tb.dut.u_prim_edn_req.u_prim_sync_reqack_data.u_prim_sync_reqack
 ANNOTATION: "VC_COV_UNR"
 Block 11 "170878399" ";"
 ANNOTATION: "VC_COV_UNR"
 Block 19 "1804254986" ";"
-CHECKSUM: "3762671585 602467306"
-INSTANCE: tb.dut.u_otp_ctrl_dai
-ANNOTATION: "VC_COV_UNR"
-Block 13 "1602293176" "state_d = ErrorSt;"
-CHECKSUM: "2261926432 1915632752"
-INSTANCE: tb.dut.u_otp_ctrl_kdi
-ANNOTATION: "VC_COV_UNR"
-Block 71 "3038555774" "state_d = DigLoadSt;"
-CHECKSUM: "714850099 3522900596"
+CHECKSUM: "739651683 2300911661"
 INSTANCE: tb.dut.gen_partitions[3].gen_buffered.u_part_buf
-Fsm error_q "3522900596"
+Fsm error_q "2300911661"
 ANNOTATION: "VC_COV_UNR"
-Transition CheckFailError->FsmStateError "6->7"
-Fsm error_q "3522900596"
+Transition FsmStateError->MacroEccCorrError "7->2"
```

Comment:
I am not sure why previous UNR did not capture this, but this is clearly an unreachable FSM transition because FsmStateError is terminal error.
Same comments applies to all the rest `Transition FsmStateError->CheckFailError "7->6"` for all partitions.

Reply:
Oh I believe this is the the `error_q` register, not the FSM state itself.
Once this gets assigned an `FsmStateError` code and the FSM goes into terminal error state, it cannot be overridden to a correctable `MacroEccCorrError` error code afterwards, because that one is not considered to be fatal.
It looks ok to me.